gradoop-data-integration/src/main/java/org/gradoop/dataintegration/importer/rdbmsimporter/connection/RegisterDriver.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.dataintegration.importer.rdbmsimporter.connection;
+
+import org.apache.log4j.Logger;
+
+import java.lang.reflect.InvocationTargetException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+/**
+ * Registers jdbc driver
+ */
+public class RegisterDriver {
+
+  /**
+   * Registers a jdbc driver
+   *
+   * @param config Database configuration
+   */
+  public static void register(RdbmsConfig config) {
+    AccessController.doPrivileged(new PrivilegedAction<Object>() {
+
+      @Override
+      public Object run() {
+        // privileged code goes here, for example:
+        Logger logger = Logger.getLogger(RegisterDriver.class);
+
+        try {
+          URL driverUrl = new URL("jar:file:" + config.getJdbcDriverPath() + "!/");
+          URLClassLoader ucl = new URLClassLoader(new URL[] { driverUrl });
+          Driver driver = (Driver) Class.forName(config.getJdbcDriverClassName(), true, ucl)
+              .getDeclaredConstructor().newInstance();
+          DriverManager.registerDriver(new DriverShim(driver));
+
+        } catch (SQLException e) {
+          System.err.println("Cannot register jdbc driver !");
```

Comment:
Please do not use any of the system println methods. You simply will never see the output in the cluster. Same goes for line 59

Reply:
Done